charts/node-red/Chart.yaml

```diff
@@ -9,7 +9,7 @@ icon: https://nodered.org/about/resources/media/node-red-icon-2.png
 
 type: application
 
-version: 0.9.0
+version: 1.0.0
 appVersion: 2.3.0
```

Comment:
Hi @Kaktor, can you chane the appVersion to 2.2.2! Must be an error from the broken chart.